frontend/src/app/task-area/task-area.component.html

```diff
@@ -1,21 +1,47 @@
 <div class="task-area">
-  <h1>All Tasks</h1>
-  <input class="new-input" type="text" placeholder="Add a new task insdie ‘All’ category" />
+  <h1>{{ currenTab }} Tasks</h1>
+
+  <input
+    (keyup.enter)="createTask()"
+    [formControl]="taskText"
+    class="new-input"
+    type="text"
+    placeholder="Add a new task insdie '{{ currenTab }}' category"
+    [class.is-invalid]="taskText.errors && (taskText.dirty || taskText.touched)"
+    [class.valid]="!taskText.errors && (taskText.dirty || taskText.touched)"
+  />
+  <label
+    style="color: red; font-weight: 300;" 
+    *ngIf="taskText.invalid && taskText.value"
+    >This filed invalid</label
+  >
 
   <ng-template #taskTemplate let-itemData="item">
-    <input type="checkbox" [checked]="itemData.status" class="custom-checkbox" id="{{ itemData.id }}" />
+    <input
+      type="checkbox"
+      (click)="checkTask(itemData.id, !itemData.status)"
```

Comment:
Can you use checkTask with itemData payload only? I mean `checkTask(itemData)`

Reply:
fixed